Makefile

```diff
@@ -0,0 +1,6 @@
+install_dependencies:
+	luarocks install lua-cjson
+	luarocks install penlight
+	luarocks install lyaml
+	luarocks install luasocket
+	luarocks install luasec
```

Comment:
Again, `luasocket` and `luasec` are unmaintained and unreliable libraries. Please do not use them.